eng/pipelines/merge-docs-main-to-live.yml

```diff
@@ -0,0 +1,86 @@
+parameters:
+- name: WhatIfPreference
+  type: boolean
+  default: false
+
+- name: DocsRepos
+  type: object
+  default:
+    - RepoName: azure-docs-sdk-java
+      RepoOwner: Azure
+    - RepoName: azure-docs-sdk-python
+      RepoOwner: MicrosoftDocs
+    - RepoName: azure-docs-sdk-node
+      RepoOwner: MicrosoftDocs
+
+- name: BaseBranch
+  type: string
+  default: live
+
+jobs:
+  - job: SyncUp
+    pool:
+      vmImage: windows-2019
+    variables:
+      DocRepoLocation: $(Pipeline.Workspace)/docs
+    steps:
+      - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+        parameters:
+          Paths: 
+          - /eng/pipelines/branch-syncup.yml
+      - ${{ each repo in parameters.DocsRepos }}:
+        - template: /eng/common/pipelines/templates/steps/sparse-checkout.yml
+          parameters:
+            SkipDefaultCheckout: true
+            Repositories:
+            - Name: ${{ repo.RepoOwner }}/${{ repo.RepoName }}
+              WorkingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+
+        - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+          parameters:
+            WorkingDirectory: $(DocRepoLocation)/${{repo.RepoName}}
+            DefaultBranchVariableName: ${{repo.RepoName}}
+
+        - pwsh: | 
+            # Fetch all branches, e.g. main, live
+            Write-Host "git -c user.name=`"azure-sdk`" -c user.email=`"azuresdk@microsoft.com`" fetch origin"
+            git -c user.name="azure-sdk" -c user.email="azuresdk@microsoft.com" fetch origin | Write-Host
+
+            # Live branch tracks remote live branch.
+            Write-Host "git checkout -b ${{ parameters.BaseBranch }} -t origin/${{ parameters.BaseBranch }}"
+            git checkout -b ${{ parameters.BaseBranch }} -t origin/${{ parameters.BaseBranch }} | Write-Host
```

Comment:
You can avoid these two commands by setting `Commitish: ${{ parameters.BaseBranch }}` in the sparse checkout step.